src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterSwitchStatements.java

```diff
@@ -27,25 +27,25 @@ String whatIsToday(Day day) {
 
     String whatIsToday2(Day day) {
         var today = "";
-        switch(day) { // violation
+        switch(day) { // violation ''switch' is not followed by whitespace'
             case MON, TUS, WED, THU, FRI -> today = "Working day";
-            default ->throw new IllegalArgumentException(); // violation
+            default ->throw new IllegalArgumentException(); // violation ''->' is not .* whitespace'
```

Comment:
Extract `new IllegalArgumentException()` to a local variable and use it instead and use a complete violation message.

Reply:
Done

Reply:
Resolved.